sensor/NmapSensor.js

```diff
@@ -213,6 +213,10 @@ class NmapSensor extends Sensor {
     return Promise.all(this.networkRanges.map((range) => {
 
       log.info("Scanning network", range, "to detect new devices...");
+      if (range.endsWith('/8')) {
+        log.info("Subnet " + range + " contains too many ip addresses to scan, skip it.")
+        return;
```

Comment:
should return a promise? otherwsie Promise.all will never return...

or better way is use async/await.